src/actions/account.js

```diff
@@ -94,66 +94,67 @@ export function handleRefreshUrl(location) {
 
 const wallet = new Wallet()
 
-export const redirectToApp = () => (dispatch, getState) => {
-   const state = getState()
-   const params = (state.account.url && (state.account.url.success_url || state.account.url.public_key)) ? `?${stringify(state.account.url)}` : ''
-   const nextUrl = state.account.url.redirect_url ? `${state.account.url.redirect_url}${params}` : `/${params}`
-   setTimeout(() => {
-      window.location = nextUrl
-   }, 1500)
+export const redirectToApp = (history) => (dispatch, getState) => {
```

Comment:
my understanding is that this can be better handled if we integrate with https://github.com/reactjs/react-router-redux

@marcinbodnar what do you think?